Dockerfile

```diff
@@ -68,27 +86,33 @@ RUN ${CONDA_EXECUTABLE} env create -f ./danesfield/deployment/conda/conda_env.ym
     ${CONDA_EXECUTABLE} clean -tipsy
 
 # Install core3d-tf_ops package from kitware-danesfield / defaults
-RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && \
-      conda clean -tipsy"]
+RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && conda activate core3d && pip install pip==20.0.1 && conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && conda clean -tipsy"]
 
 # Install opencv package from conda-forge
-RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      conda install -c conda-forge -y opencv && \
-      conda clean -tipsy"]
+RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && conda activate core3d && conda install -c conda-forge -y opencv && conda clean -tipsy"]
+
+RUN chmod +x pipeline.sh && \
+  chmod +x /ColmapForVisSat/ubuntu1804_install_dependencies.sh && \
+  chmod +x /ColmapForVisSat/ubuntu1804_install_colmap.sh && apt-get update && \
+  /ColmapForVisSat/ubuntu1804_install_dependencies.sh && \ 
+  cd /ColmapForVisSat && ./ubuntu1804_install_colmap.sh
+
+RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && conda activate core3d && pip install -r /VisSatSatelliteStereo/requirements.txt && export CPLUS_INCLUDE_PATH=/usr/include/gdal && export C_INCLUDE_PATH=/usr/include/gdal && pip install GDAL==$(gdal-config --version) --global-option=build_ext --global-option=\"--include-dirs=/usr/include/gdal\""]
+
+#RUN cd /filter && \
+# mkdir build && \
+# cd build && \
+# cmake -DCMAKE_BUILD_TYPE=Release .. && \
+# make
+
+RUN git clone https://github.com/LAStools/LAStools.git && \
+  cd LAStools && \
+  make
 
 # Install Danesfield package into CORE3D Conda environment
 COPY . ./danesfield
 RUN rm -rf ./danesfield/deployment
-RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      pip install --upgrade pip && \
-      pip install -e ./danesfield"]
+RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && conda activate core3d && pip install -e ./danesfield"]
 
 # Set entrypoint to script that sets up and activates CORE3D environment
-ENTRYPOINT ["/bin/bash", "./danesfield/docker-entrypoint.sh"]
-
-# Set default command when executing the container
-CMD ["/bin/bash"]
```

Comment:
I think we probably want to leave this CMD call in unless there's a compelling reason to leave it off